src/nprintml/label/aggregator/pcap.py

```diff
@@ -11,7 +11,15 @@ class PcapLabelAggregator(LabelAggregator):
     size, to run autoML on the generated samples.
 
     """
-    def __init__(self, npt_csv, label_csv):
+    def __init__(self, label_csv):
+        super().__init__(label_csv)
+        self.labels = self.load_label(label_csv)
```

Comment:
And unlike the index aggregator, the pcap aggregator extends its initializer to set `labels` to the eagerly-loaded labels.